overlord/snapstate/snapstate_test.go

```diff
@@ -1122,20 +1126,15 @@ func (s *snapmgrTestSuite) TestRemoveDiskSpaceForSnapshotNotCheckedWhenSnapshots
 	})
 
 	_, err := snapstate.Remove(s.state, "some-snap", snap.R(0), nil)
-	c.Assert(err, IsNil)
+	c.Check(err, IsNil)
 	c.Assert(automaticSnapshotCalled, Equals, true)
 }
 
-func (s *snapmgrTestSuite) TestRemoveDiskSpaceCheckDisabled(c *C) {
+func (s *snapmgrTestSuite) TestRemoveDiskSpaceForSnapshotNotCheckedWhenSnapshotsDisabled(c *C) {
```

Comment:
this and TestRemoveDiskSpaceCheckDisabled seems to be the same test ? what am I missing?

Reply:
ah, one disables the feature explicitly and one uses the default?

Reply:
I've refactored these tests separately in #9274